### ISSUE TITLE
app.js not building for windows users

### ISSUE THREAD

**Issue by pcavagna** [synapsestudios/lively#124]
## Description

After cloning a fresh copy of lively app.js not building for windows users, making the site inaccessible.  Works fine for mac users.

```
GET http://localhost:9001/js/app.js     (index):20 
```

```
  </head>
    <body>
        <script src="/js/app.js"></script>
```

**Comment by paulstatezny** [comment]
Webpack? :arrow_double_up:

**Comment by bobeagan** [comment]
:snowman:

**Comment by paulstatezny** [comment]
Should be fixed by #125

**Comment by pcavagna** [comment]
:+1:
